components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__family.xml

```diff
@@ -113,26 +228,30 @@
 {{include reference="PhenoTips.TabelarDataMacros" /}}
 
 {{velocity}}
+$xwiki.ssx.use('PhenoTips.UIX_Field__family')##
+
 #set ($targetClassName = 'PhenoTips.RelativeClass')
 #if ($xcontext.action == 'edit' || $doc.getObject($targetClassName))
-  {{error}}This component is deprecated. Please specify relevant family relationships in the pedigree.{{/error}}
 
   (% class="clear relatives-info" %)(((
+    {{error}}This component is deprecated. Please specify relevant family relationships in the pedigree.{{/error}}
```

Comment:
doesn't matter, component is dperecated and will only be enabled temporarily to check migrations. Marta added it without translation so I'm fine with it for sure